core/include/irods/private/http_api/openid.hpp

```diff
@@ -14,7 +14,7 @@
 
 namespace irods::http::openid
 {
-	auto create_oidc_request(boost::urls::url_view _url)
+	auto create_oidc_request(const boost::urls::url_view& _url)
```

Comment:
According to the Boost docs, `url_view` functions like a `string_view`. My understanding is that passing by value is favored for `string_view`s over passing by reference. Maybe it doesn't matter so much since this is a "private" function and doesn't need to be particularly optimized...

Reply:
For reference: https://www.boost.org/doc/libs/1_83_0/libs/url/doc/html/url/ref/boost__urls__url_view.html#url.ref.boost__urls__url_view.description

Reply:
Yeah, I agree with you to treat this as a string_view. Just added this change to address clang-tidy.

Revert and add lint ignore?

Reply:
Yes please.

Reply:
Done.